editoast/editoast_schemas/src/train_schedule/train_schedule_options.rs

```diff
@@ -14,8 +14,22 @@ pub struct TrainScheduleOptions {
     #[derivative(Default(value = "true"))]
     #[serde(default = "default_use_electrical_profiles")]
     use_electrical_profiles: bool,
+
+    #[derivative(Default(value = "true"))]
```

Comment:
```suggestion
    #[derivative(Default(value = "false"))]
```

We keep this flag to false for review purpose, but before merging, we need to set it to `false`